bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -8,6 +8,18 @@ metadata:
     scheduler: "true"
     revision: "REVISION"
 spec:
+  initContainers:
+  - name: wait-for-host-port
+    image: ${IMAGE}
+    imagePullPolicy: IfNotPresent
+    command: ['/usr/bin/timeout', '105', '/bin/bash', '-c'] # a bit more than 60s for graceful termination + 35s for minimum-termination-duration, 5s extra cri-o's graceful termination period
```

Comment:
this is far too much for kube-scheduler. It does not hurt much, but still. At least the comment is wrong :)